src/ethereum_test_tools/spec/eof/eof_test.py

```diff
@@ -267,8 +266,10 @@ class EOFStateTest(EOFTest):
 
     tx_gas_limit: int = 10_000_000
     tx_data: Bytes = Bytes(b"")
+    tx_sender_funding_amount: int = 1_000_000_000_000_000_000_000
     env: Environment = Field(default_factory=Environment)
     container_post: Account = Field(default_factory=Account)
+    pre: Alloc | None = None
```

Comment:
This is just a necessary trick to satisfy mypy:

![image](https://github.com/ethereum/execution-spec-tests/assets/11726710/129f359f-0384-4dfd-bb88-0f5c8e218010)

Since the `pre` is actually simply filled by the test filler automatically if not set in the test, but if set as a required parameter, mypy complains.